pkg/api/v1/source.go

```diff
@@ -0,0 +1,54 @@
+///////////////////////////////////////////////////////////////////////
+// Copyright (c) 2017 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+///////////////////////////////////////////////////////////////////////
+
+package v1
+
+import (
+	strfmt "github.com/go-openapi/strfmt"
+
+	"github.com/go-openapi/errors"
+	"github.com/go-openapi/swag"
+)
+
+// NO TESTS
+
+// Source source
+// swagger:model Source
+type Source struct {
```

Comment:
We talked off-line and I assume that the URL is actually going in the Function object to point to the Source entity which is just the code?